### ISSUE TITLE
Set config variables

### ISSUE THREAD

**Issue by noamross** [ropensci/tesseract#6]
On the command line, tesseract lets one set config variables that otherwise have to be set in the config file of the library.  I'm not sure where this lives in the C++ API, but would it be possible allow setting these variables in `tesseract()`?  My specific application is a character whitelist, which on the command line would be

`tesseract stdin stdout -c tessedit_char_whitelist=abcdefghijklmnopqrstuvwxyz`

I'm thinking something like 

`ocr("file.tiff", tesseract(options=list(tessedit_char_whitelist="abcdefghijklmnopqrstuvwxyz0123456789")))`

**Comment by jeroenooms** [comment]
Did you see https://github.com/ropensci/tesseract/issues/5 ?\

**Comment by noamross** [comment]
*headslap*.  Sorry, thanks!  Closing...

**Comment by jeroenooms** [comment]
Does it work? I havent tested it properly yet.

**Comment by noamross** [comment]
Yes, I was able to whitelist only ASCII characters for some docs I scanned.

On Sun, Nov 20, 2016, 4:03 AM Jeroen Ooms <notifications@github.com> wrote:

Does it work? I havent tested it properly yet.

—
You are receiving this because you modified the open/close state.


Reply to this email directly, view it on GitHub
<https://github.com/ropensci/tesseract/issues/6#issuecomment-261766838>,
or mute
the thread
